sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -319,6 +320,9 @@ static const struct snd_soc_dapm_route sof_map[] = {
 	/* other jacks */
 	{ "IN1P", NULL, "Headset Mic" },
 
+	/* digital mics */
+	{"DMic", NULL, "SoC DMIC"},
+
```

Comment:
Don't we need this for other drivers, e.g. skl_hda_generic when DMICs are used?

Reply:
Actually this skl_hda_generic is missing a dependency on SND_SOC_DMIC as well in the Kconfig